.github/workflows/deploy.yml

```diff
@@ -59,7 +59,7 @@ jobs:
             --minify \
             --baseURL "${{ steps.pages.outputs.base_url }}/"
       - name: Upload artifact
-        uses: actions/upload-pages-artifact@v2
+        uses: actions/upload-pages-artifact@v3
```

Comment:
https://github.com/altive/handbook/actions/runs/13147588015/job/36688963512
> Error: This request has been automatically failed because it uses a deprecated version of `actions/upload-artifact: v3`. Learn more: https://github.blog/changelog/2024-04-16-deprecation-notice-v3-of-the-artifact-actions/

上記に対応するためにバージョンをあげました！
v3で v4 upload-artifact に対応したようです✍️

https://github.com/actions/upload-pages-artifact/releases/tag/v3.0.0